content/blog/2020-05-rules-0-0-1/index.Rmarkdown

```diff
@@ -0,0 +1,122 @@
+---
+title: rules 0.0.1
+slug: rules-0-0-1
+description: >
+    The rules package is on CRAN and enables rule-based models for tidymodels.
+date: 2020-05-18
+author: Max Kuhn
+photo:
+  url: https://unsplash.com/photos/lu15z1m_KfM
+  author: Lili Popper
+categories:
+  - package
+---  
```

Comment:
```suggestion
  - package
tags:
  - rules
  - tidymodels
---  
```

Reply:
For tags we're adding the org (tidyverse, r-lib, tidymodels, and the name of package(s) in the post).